intel-extension-for-pytorch/Distributed/README.md

```diff
@@ -0,0 +1,66 @@
+# Intel® Extension for PyTorch with KubeRay on RedHat® OpenShift
+
+In this demo notebook we demonstrate how you can utilize the Intel® Extension for PyTorch to run distributed pytorch workloads on RedHat® OpenShift. The notebook uses the [CodeFlare SDK](https://github.com/project-codeflare/codeflare-sdk) to connect to the [KubeRay](https://docs.ray.io/en/latest/cluster/kubernetes/index.html) operator and CodeFlare operator to launch distributed jobs. Before we start running the notebook we need to setup the environment correctly to run it successfully. More details can be found in the section below.
+
+## Setup
+Before we start running the notebook we need to make sure a few things are in place that are described in this section.
+
+* Install the [OpenShift AI](https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-ai) operator. Follow the steps below to install the OpenShift AI operator correctly. If you already have OpenShift AI operator installed you can skip to Step 2.
+
+    - Open the OpenShift Web Console and click on the Operator->OperatorHub tab on the left panel. Refer to the screenshot below.
+        ![Step-1](./assets/OperatorHub.png)
+    - Now in the search box type "OpenShift AI" and that'll show you the "RedHat OpenShift AI" tile for the operator. Refer to the screenshot below.
+        ![Step-2](./assets/RHODS-operator.png)
+    - Click on the tile and click on the Install button. This will install all the necessary resources and the operator.
```

Comment:
`Click on the tile and click on` -> `Select the tile and click on`

Reply:
Updated